### ISSUE TITLE
Case Study Page / Stories Page

### ISSUE THREAD

**Issue by valentinajemuovic** [atomiv/atomiv.github.io#69]

**Comment by valentinajemuovic** [comment]
@TamaraG13, the current "Why" page was hidden from the menu and it will be later transformed into Case Study page so that it can be longer text (and fit the challenge - solution form). We'll discuss the case study page in the next version.

**Comment by TamaraG13** [comment]
Yes, I see that. Is that mean we will not have "Why" page at all? 
I think it is good to have it, we can just write additional content for that or for the case study, what do you think?
